recruiter/tests.py

```diff
@@ -147,3 +179,24 @@ def test_new_recruiter_account_with_valid_data(valid_recruiter_data, client):
     assert example_recruiter_from_db.email == valid_recruiter_data["email"]
     assert example_recruiter_from_db.name == valid_recruiter_data["name"]
     assert example_recruiter_from_db.phone_number == valid_recruiter_data["phone_number"]
+
+
+@pytest.mark.django_db
+def test_recruiter_my_jobs_loads_correct_data(example_recruiter, example_job_application, client):
+    recruiter_object = Recruiter.objects.get(name="a")
+    client.force_login(recruiter_object.user)
+    response = client.get("/myjobs")
+    assert response.status_code == 200
+    jobs_and_applications_from_response = response.context["jobs_and_applications"]
+    assert list(jobs_and_applications_from_response.keys()) == list(Job.get_jobs_by_recruiter_id(recruiter_object))
+    assert all(
+        list(jobs_and_applications_from_response[job]) == list(Application.get_applications_by_job(job)) for job in
+        jobs_and_applications_from_response)
+
+
+@pytest.mark.django_db
+def test_user_that_is_not_a_recruiter_cannot_access_my_jobs_url(example_student, client):
+    response = client.get("/myjobs")
+    assert response.status_code == 404
+    client.force_login(example_student.user)
```

Comment:
What is a point of doing a force_login without also doing an HTTP request? 
Please try to limit each test to a single HTTP call.

Reply:
My bad I forgot to make another request.
I'll separate it into two tests.

Reply:
done